### DISCUSSION TITLE
More CLI commands for users e.g. Unlock a user

### DISCUSSION THREAD

**Discussion by bymayo** [craftcms/cms#12345] [Ideas] [answered]
We had a situation where some how our admin account got locked out of a site, and the only way we could fix this was to manually change the value in the db.

Would be nice to have more `./craft users/` commands e.g. `./craft users/unlock --email=john@website.com` so this could be done a bit safer.

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 3.8 and 4.4 via #12511 🎉

**Reply by brandonkelly** [reply]
4.4 is out now!
